### DISCUSSION TITLE
log_discarded_spans is on main not in v2.6.1

### DISCUSSION THREAD

**Discussion by shalk** [grafana/tempo#4300] [General]
I found a new config item `log_discarded_spans` in main branch. The feature is added three month ago.

But I do not found the config item in v2.6.1 whichi is release two weeks ago

I want to confirm the log_discarded_spans is since v2.6.2 or something

**Comment by javiermolinar** [comment]
This will be in v2.7. There are no plans to release a v2.6.2
